src/main/java/com/labsynch/labseer/chemclasses/rdkit/BBChemStructureService.java

```diff
@@ -0,0 +1,338 @@
+		String requestString = requestData.toString();
+		logger.info("requestString: " + requestString);
+		String postResponse = SimpleUtil.postRequestToExternalServer(url, requestString, logger);
+		logger.info("Got response: "+ postResponse);
+
+		// Parse the response json
+		ObjectMapper responseMapper = new ObjectMapper();
+		JsonNode responseNode = responseMapper.readTree(postResponse);
+		return responseNode.asText();
+	}
+
+	@Override
+	public List<RDKitStructure> getRDKitStructuresFromSDFService(String molfile) throws CmpdRegMolFormatException {
+		List<RDKitStructure> rdkitStructures = new ArrayList<RDKitStructure>();
+
+		// Read the preprocessor settings as json
+		JsonNode jsonNode = null;
+		try{
+			jsonNode = getPreprocessorSettings();
+		} catch (IOException e) {
+			logger.error("Error parsing preprocessor settings json");
+			throw new CmpdRegMolFormatException("Error parsing preprocessor settings json");
+		}
+
+		// Extract the processURL
+		JsonNode urlNode = jsonNode.get("parseURL");
+		if(urlNode == null || urlNode.isNull()) {
+			logger.error("Missing preprocessorSettings parseURL!!");
+		}
+		String url = urlNode.asText();
+
+		// Get the standardization actions and options
+		ObjectMapper mapper = new ObjectMapper();
+		ObjectNode requestData = mapper.createObjectNode();
+
+		// Add the structures to the request
+		Base64.Encoder encoder = Base64.getEncoder();
+		String encodedString = encoder.encodeToString(molfile.getBytes(StandardCharsets.UTF_8) );
+		requestData.put("molv3", encodedString);
+
+		// Post to the service and parse the response
+		try {
+			String requestString = requestData.toString();
+			logger.info("requestString: " + requestString);
+			String postResponse = SimpleUtil.postRequestToExternalServer(url, requestString, logger);
+			logger.info("Got response: "+ postResponse);
+
+			// Parse the response json to get the standardized mol
+			ObjectMapper responseMapper = new ObjectMapper();
+			JsonNode responseNode = responseMapper.readTree(postResponse);
+			for (JsonNode responseJsonNode : responseNode)  {
+				RDKitStructure rdkitStructure = new RDKitStructure();
+				JsonNode molNode = responseJsonNode.get("mol");
+				rdkitStructure.setMol(molNode.asText());
+
+				rdkitStructure.setProperties(new HashMap<String, String>());
+				JsonNode propertiesJsonNode = responseJsonNode.get("properties");
+				for (JsonNode propertyNode : propertiesJsonNode) {
+					String name = propertyNode.get("name").asText();
+					String value = propertyNode.get("value").asText();
+					rdkitStructure.getProperties().put(name, value);
+				}
+				rdkitStructures.add(rdkitStructure);
+			}
+
+
+		} catch (Exception e) {
+			logger.error("Error posting to preprocessor service: " + e);
+			throw new CmpdRegMolFormatException("Error posting to preprocessor service: " + e);
+		}
+
+		return rdkitStructures;
+
+	}
+
+	@Override
+	public RWMol getPartialiallySanizedRWMol(String molstructure) {
```

Comment:
nit: Name has two typos to be fixed "Partialially" -> "Partially" and "Sanized" -> "Sanitized"